src/components/skills-quiz/__mocks__/react-instantsearch-dom.jsx

```diff
@@ -6,40 +6,57 @@ const MockReactInstantSearch = jest.genMockFromModule(
   'react-instantsearch-dom',
 );
 
-const fakeHits = [
-  {
-    objectID: '1',
-    title: 'bla',
-    key: 'Bees101',
-    type: 'course',
-    aggregation_key: 'course:Bees101',
-    authoring_organizations: [],
-    card_image_url: 'https://fake.image',
-    course_keys: [],
-  },
-  {
-    objectID: '2',
-    title: 'blp',
-    key: 'Wasps200',
-    type: 'course',
-    aggregation_key: 'course:Wasps200',
-    authoring_organizations: [],
-    card_image_url: 'https://fake.image',
-    course_keys: [],
-  },
-];
-
-MockReactInstantSearch.configure = {
+const originalState = {
+  hits: [
+    {
+      objectID: '1',
+      title: 'bla',
+      key: 'Bees101',
+      type: 'course',
+      aggregation_key: 'course:Bees101',
+      authoring_organizations: [],
+      card_image_url: 'https://fake.image',
+      course_keys: [],
+    },
+    {
+      objectID: '2',
+      title: 'blp',
+      key: 'Wasps200',
+      type: 'course',
+      aggregation_key: 'course:Wasps200',
+      authoring_organizations: [],
+      card_image_url: 'https://fake.image',
+      course_keys: [],
+    },
+  ],
   nbHits: 2,
 };
 
+let mockState = {
```

Comment:
nit: this could be a `const`, as `resetMockReactInstantSearch` could just override the properties without re-defining the variable, similar to `setFakeHits`.